src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/basic/PortalSpawnModifier.java

```diff
@@ -55,11 +58,41 @@ public void portalSpawn(CreatureSpawnEvent event) {
 				event.setCancelled(true);
 				return;
 			}
-			if (roll(ghastChance)) {
+
+			//Disabled since we don't want ghasts spawning in nether portals.
+			//if (roll(ghastChance)) {
+			//	event.setCancelled(true);
+			//	spawnGhast(event.getLocation());
+			//}
+		}
+	}
+
+	//Wither Skeleton + Ghast spawns
+	@EventHandler
+	private void hellSpawn(CreatureSpawnEvent event) {
+		World world = event.getLocation().getWorld();
+		if (event.getSpawnReason() == SpawnReason.NETHER_PORTAL) {
+			return;
+		}
+
+		if (event.getEntityType() == EntityType.ZOMBIFIED_PIGLIN) {
+			if (roll(witherSkeletonChance)) {
+				//If witherSkeletonChance rolls true, stop the pigman from spawning, and spawn a wither skeleton in its place.
+				event.setCancelled(true);
+				world.spawnEntity(event.getLocation(), EntityType.WITHER_SKELETON);
+			} else if (roll(ghastChance)) {
+				//If ghastChance rolls true, stop pigman from spawning, and spawn ghast 2 blocks higher.
 				event.setCancelled(true);
 				spawnGhast(event.getLocation());
 			}
 		}
+
+		if (event.getEntityType() == EntityType.WITHER_SKELETON && event.getSpawnReason() == SpawnReason.CUSTOM) {
```

Comment:
Should be configurable

Reply:
this is configurable, lines 170, and 171 of the default config have chances. if you want it disabled, you simply set it to 0, or disable the hack entirely.